<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportVideoDialog</class>
 <widget class="QDialog" name="ExportVideoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>413</width>
    <height>145</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExportVideoDialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>391</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>导出路径</string>
   </property>
   <widget class="QLineEdit" name="exportDestEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>31</y>
      <width>260</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>260</width>
      <height>0</height>
     </size>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="selectFolderButton">
    <property name="geometry">
     <rect>
      <x>274</x>
      <y>30</y>
      <width>107</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>107</width>
      <height>20</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>107</width>
      <height>32</height>
     </size>
    </property>
    <property name="text">
     <string>选择路径</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,0">
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textDirection">
       <enum>QProgressBar::TopToBottom</enum>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="startBtn">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>开始</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="closeBtn">
      <property name="minimumSize">
       <size>
        <width>80</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>80</width>
        <height>32</height>
       </size>
      </property>
      <property name="text">
       <string>Close</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>closeBtn</sender>
   <signal>clicked()</signal>
   <receiver>ExportVideoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>359</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
